docs/deployment.md

```diff
@@ -14,7 +14,23 @@ To deploy to production server:
   git submodule init && git submodule update
   ```
   Repository details see [Project repository](git-repository.md).
-3. Link production configuration for docker-compose, Details see [Docker container bootstrapping](../README.md#initializationdocker-container-bootstrapping).
+3. a. Link production configuration for docker-compose, Details see [Docker container bootstrapping](../README.md#initializationdocker-container-bootstrapping).
+   b. Edit the `compose-production.yml` file and change/add environment settings for celery and webapp. 
+      List of available additional settings (no quotes for strings!):
+   
+       ```
+        - DJANGO_EMAIL_HOST=<your smtp>   # ie. smtp.example.com
```

Comment:
* You mean 'e.g.' (for example), not 'i.e.' (it is), don't you?
* I'd replace the placeholder by the example and move the explanation within the placeholder to the comment:

        - DJANGO_EMAIL_HOST=smtp.example.com   # The SMTP server to use for sending mails to users

    That way, you avoid ambiguity in the syntax, e.g. someone thinking they'd have to use

        - DJANGO_EMAIL_HOST=<mail.my.example.com>
    with literal `<` and `>` when `mail.my.example.com` is their server's domain.